insights/specs/default.py

```diff
@@ -548,6 +548,8 @@ def semid(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovirt_engine_boot_log = simple_file("/var/log/ovirt-engine/boot.log")
     ovirt_engine_console_log = simple_file("/var/log/ovirt-engine/console.log")
+    ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
+    ovs_appctl_fdb_show_bridge = foreach_execute(ovs_vsctl_list_br, "/usr/bin/ovs-appctl fdb/show %s")
```

Comment:
Does `fdb/show` take the multi-line output of `list bridge` as input?  That doesn't seem to match what you are trying to accomplish.

Reply:
@bfahr - Yes, `fdb/show` takes the multi-line output of `ovs-vsctl list-br` as input. Here, using spec 
`ovs_vsctl_list_br`, all the bridge names are collected and later passed to spec `ovs_appctl_fdb_show_bridge`. Following is the sample output of insights-cat:

(.python3) # insights-cat ovs_vsctl_list_br
```
CommandOutputProvider("'/usr/bin/ovs-vsctl list-br'")
br-int
br-int1
br-tun
br0
```

(.python3) # insights-cat ovs_appctl_fdb_show_bridge
```
CommandOutputProvider("'/usr/bin/ovs-appctl fdb/show br-int'")
 port  VLAN  MAC                 Age
    1    29  MAC1                  270
    1    29  MAC2                  266
    1   100  MAC3                  263
    1    29  MAC4                  234
CommandOutputProvider("'/usr/bin/ovs-appctl fdb/show br-int1'")
 port  VLAN  MAC                 Age
    1    29  MAC1                  270
    1    29  MAC2                  266
    1   100  MAC3                  263
    1    20  MAC4                  254
    1    20  MAC5                  253
    1    29  MAC6                  252
    1    30  MAC7                  251
    1    29  MAC8                  250
CommandOutputProvider("'/usr/bin/ovs-appctl fdb/show br-tun'")
 port  VLAN  MAC                 Age
    1    29  MAC1                  270
CommandOutputProvider("'/usr/bin/ovs-appctl fdb/show br0'")
 port  VLAN  MAC                 Age
```